pkg/common/cns-lib/vsphere/utils.go

```diff
@@ -396,14 +396,9 @@ func GetTagManager(ctx context.Context, vc *VirtualCenter) (*tags.Manager, error
 // managed datastores of given VC cluster.
 // The 1st output parameter will be shared datastores.
 // The 2nd output parameter will be vSAN-direct managed datastores.
-func GetCandidateDatastoresInCluster(ctx context.Context, vc *VirtualCenter, clusterID string) (
-	[]*DatastoreInfo, []*DatastoreInfo, error) {
+func GetCandidateDatastoresInCluster(ctx context.Context, vc *VirtualCenter, clusterID string,
+	includevSANDirectDatastores bool) ([]*DatastoreInfo, []*DatastoreInfo, error) {
```

Comment:
Can update method doc for GetCandidateDatastoresInCluster in regard to new parameter `includevSANDirectDatastores`?

Reply:
Done